samples/invalid.stm

```diff
@@ -0,0 +1,13 @@
+Valid
+unknown 1 UnknownSpeaker 0.00 3.20 <o,f0,male> testing testing one two three or maybe 10 4
+unknown chan UnknownSpeaker 0.00 3.20 <o,f0,male> testing testing one two three
+unknown 2 UnknownSpeaker 0.00 3.20 <o,f0,male> testing testing, one two three!
```

Comment:
if you had an invalid line up here (on line2, line3 or line4), would the `convert()` def stop at the first invalid line?

should we check for valid lines in the entire file or do we just stop converting at the first invalid occurrence and then come back to the file?

Reply:
Yup, the line order doesn't matter so I separated them for ease of understanding.